CHANGES.rst

```diff
@@ -31,6 +31,8 @@ Specviz2d
 API Changes
 -----------
 
+- New plugin-level ``open_in_tray`` method to programmatically show the plugin. [#1559]
```

Comment:
Hmm, this is a new API, so should be under "New Features" and not "API Changes"?

Reply:
I never know in cases like this, but I guess that makes sense, so I'll move it!